app/components/mandatarissen/mandataris-extra-info-card.hbs

```diff
@@ -43,6 +43,7 @@
 >
   <div class="au-o-box">
     <Form::Instance
+      @showHistoryDescription={{true}}
```

Comment:
i would make true the default

Reply:
I actually thought you would say it ;p I'll change it!